src/tests/SearchView.test.jsx

```diff
@@ -396,27 +449,33 @@ describe('Search View', () => {
         }),
       ),
     );
-    const user = userEvent.setup();
+    const user = userEvent.setup({ delay: null });
 
     render(<SearchView />);
 
     const searchInput = getByRole('textbox');
     await user.type(searchInput, 'terryjones@python.com');
 
-    await screen.findByRole('button', { name: 'repository' });
+    await waitFor(() =>
+      expect(queryByText('Please select a repository')).toBeInTheDocument(),
+    );
```

Comment:
I believe `await screen.findByText('Please select a repository')` would be useful here as well.

Reply:
I believe I tried that initially. This is the error I get if I replace that line:
```
received value must be an HTMLElement or an SVGElement.
    Received has type:  object
    Received has value: {}
```

Reply:
if I replace that line here, is it redundant to then `expect` that text `toBeInTheDocument`?

Reply:
Yeah I believe so.
You can use `expect(await findByText(...)).toBeInTheDocument()` if you want to be extra explicit :-)
The await is important so that the result of the find is expected.